source/containers.tex

```diff
@@ -10082,7 +10082,8 @@
 \tcode{first}
 and
 \tcode{last})
-and no reallocations if iterators \tcode{first} and \tcode{last} are of forward, bidirectional, or random access categories.
+and no reallocations if
+\tcode{InputIterator} meets the \oldconcept{ForwardIterator} requirements.
```

Comment:
I'm not sure whether the change in these lines is desired or editorial. It seems that the old wording can also be interpreted as "[...] if `InputIterator` meets the _Cpp17ForwardIterator_ requirements <ins>**or models `forward_iterator`**</ins>".